addon/components/item-chooser/component.js

```diff
@@ -0,0 +1,49 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+  layout,
+  classNames: ['item-chooser'],
+
+  // Options
+  placeholder: 'Pick a list',
+  recordType: 'list',
+  createOptionPlaceholder: 'Create <strong>#item#</strong> list',
```

Comment:
may be we can start with: `createOptionPlaceholder: '<strong>#item#</strong>'` ?